docs/information/zigbee_network.md

```diff
@@ -29,3 +29,51 @@ Zigbee2MQTT logs the device type of your devices on startup, e.g.:
+
+### Zigbee2MQTT
+
+The Zigbee stack has a certain amount of home automation protocol already defined; as discussed, devices can be formed into groups and scenes defined, switches can be bound to those groups.  In such a setup the Zigbee controller might help configure the network, but afterwards it is passive at the application level.
+
+If more flexibility is required than comes in the pre-defined Zigbee clusters, for example “Turn on the fan when the relative humidity is over 70%.”, then you need more clever control.  It is here that Zigbee2MQTT comes in.  It translates between Zigbee and MQTT.
+
+When a device is added to the network with a Zigbee-Herdsman controller, the controller uses the low-level configuration clusters to interview the device and find out what the device is, what endpoints it has, and what clusters each of those endpoints implements.  The Zigbee-Herdsman-Converters then record, for each model of device, which clusters the controller should bind to, and how the conversion to MQTT should be handled.  Most devices in Zigbee-Herdsman-Converters use generic converters that bind to each Zigbee cluster and provide a standard MQTT interface for that cluster.
+
+With this setup, when a switch is activated, it sends a message to the Zigbee2MQTT controller.  The controller then sends out an MQTT message.  The MQTT controller (which is different to the zigbee controller.  e.g. the Home Assistant package) then decides what to do based on that message.  It might, for example, decide to turn on a particular light, so it would send an MQTT message requesting the light to turn on.  Zigbee2MQTT would receive that message, then send a zigbee message to the light’s endpoint using the appropriate zigbee cluster.
+
+This system is significantly more flexible than the base Zigbee system.  But it is also higher latency (it takes longer for the system to react to a switch being toggled) and it has more points of failure.  With the base Zigbee setup, not even the controller is involved once setup is complete.  With the Zigbee2MQTT setup there are two zigbee messages, two MQTT messages, and three extra processing steps (the main controller deciding what to do, and MQTT processing the messages in each direction).
+
+In addition, Zigbee2MQTT currently binds to each endpoint in the network.  Many of these endpoints will broadcast state information periodically to anything bound to them.  By binding to every device in the network, whether its state is relevant or not, zigbee2MQTT is causing a bunch of traffic on the zigbee network that would otherwise not be there.  This limits the practical size of the network.
```

Comment:
- What do you mean with "binding" here?
- > zigbee2MQTT is causing a bunch of traffic on the zigbee network that would otherwise not be there
 - Could you explain this?

Reply:
I have some ML-ST-D200 switch adapters.  If I look in database.db I see: "binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"controller","endpointID":1},{"cluster":8,"type":"endpoint","deviceIeeeAddress":"controller","endpointID":1}]  (I've replaced the MAC address of my controller with the word controller to make it clearer).

This looks like the converters in zigbee-herdsman-converters are telling zigbee-herdsman to connect (bind) to various endpoints/clusters on the switch so they they can listen to state change messages.

Those devices are sending their state back to the controller periodically, even when their state isn't changing (i.e. not just when you've changed the state).  The period doesn't seem to be fixed - I think it backs off when the state hasn't changed for a while.  But just after a state change they're quite chatty.

I'm looking to get about 50 of these, and I'm a little concerned about network chatter.  I haven't tested it yet though.  My plan was to modify devices.yaml to add a "passive: true" setting which stopped zigbee-herdsman from making those bindings.

That was going to happen in my spare time though... so perhaps by 2022?

But we could take that section out of this text.

Reply:
By default no bindings are made, are you using the  report feature? https://www.zigbee2mqtt.io/information/report.html

Reply:
No, I'm not using report.  Why don't we drop the last paragraph of this text.  I'll dig into where those binds are coming from.

Is there anything else you think should change?

Reply:
I added a different paragraph at the end comparing with Phillips Hue.

Reply:
looks good, thanks!